packages/@aws-cdk/aws-rds/test/cluster.test.ts

```diff
@@ -740,7 +800,7 @@ describe('cluster', () => {
     });
 
     // THEN
-    expect(() => cluster.addRotationSingleUser()).toThrow(/without secret/);
+    expect(() => cluster.addRotationSingleUser()).toThrow(/without master secret/);
```

Comment:
Let's re-format this while we're in the area:

```suggestion
    expect(() => {
      cluster.addRotationSingleUser();
    }).toThrow(/A single user rotation for master secret was already added to this cluster/);
```